notebooks/00_quick_start/sar_movieratings_with_azureml_designer.ipynb

```diff
@@ -0,0 +1,326 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quickstart to integrate Recommenders in AzureML Designer\n",
+    "\n",
+    "This notebook shows how to integrate any algorithm in Recommenders library into AzureML Designer. \n",
+    "\n",
+    "[AzureML Designer](https://docs.microsoft.com/en-us/azure/machine-learning/concept-designer) lets you visually connect datasets and modules on an interactive canvas to create machine learning models. \n",
+    "\n",
+    "![img](https://recodatasets.blob.core.windows.net/images/designer-drag-and-drop.gif)\n",
+    "\n",
+    "One of the features of AzureML Designer is that it is possible for developers to integrate any python library to make it available as a module. In this notebook are are going to show how to integrate [SAR](sar_movielens.ipynb) and several other modules in Designer\n",
+    "\n",
+    "\n",
+    "## Installation\n",
+    "\n",
+    "The first step is to install AzureML Designer SDK. Assuming that you have installed the Recommenders environment `reco_base` as explained in the [SETUP.md](../../SETUP.md), you need to install:\n",
+    "```bash\n",
+    "conda activate reco_base\n",
+    "pip install keyring artifacts-keyring\n",
+    "pip install azureml-designer-tools==0.1.10.post8799295 --extra-index-url=https://msdata.pkgs.visualstudio.com/_packaging/azureml-modules%40Local/pypi/simple/ \n",
```

Comment:
We published new version of register tool. It supports local folder as module source now. You can try it by: pip install azureml-designer-tools==0.1.18 --extra-index-url=https://msdata.pkgs.visualstudio.com/_packaging/azureml-modules%40Local/pypi/simple/

Reply:
I'll update to the newest version